llvm/utils/mlgo-utils/mlgo/corpus/extract_ir.py

```diff
@@ -20,8 +20,8 @@
 
 To change the logging verbosity, pass an integer representing the desired
 verbosity to the --verbosity flag. Use 0 for all logs, status information,
-and detailed debug information, -1 for solely warnings, and -2 to not produce
-any output.
+and detailed debug information, 30 for solely warnings, and 50 or higher to
```

Comment:
why not leave the flag values to 0, -1, -2 and then map them on line 128 accordingly?

Reply:
Let me put it differently: the documentation says there are really 3 settings: 
- very verbose
- warnings
- quiet

If not 0, -1, -2, how about an enum flag with those values?

Reply:
Switched to supporting textual representations.